pyproject.toml

```diff
@@ -3,6 +3,19 @@ name = "shared-memory-dict"
 version = "0.1.0"
 description = "A very simple shared memory dict implementation"
 authors = ["Arquitetura LuizaLabs <arquitetura@luizalabs.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/luizalabs/shared-memory-dict"
+repository = "https://github.com/luizalabs/shared-memory-dict"
+keywords = ["django", "aiocache", "shared memory"]
```

Comment:
Does poetry supports `python_requires>=3.8` here? [In setup.py](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) it adds metadata to restrict the versions of python that can install this package.

Reply:
Works like a charm 🖤 
https://test.pypi.org/project/shared-memory-dict/